src/repr/src/explain/mod.rs

```diff
@@ -104,6 +105,14 @@ impl fmt::Display for ExplainError {
             ExplainError::SerdeJsonError(error) => {
                 write!(f, "{}", error)
             }
+            ExplainError::LinearChainsPlusRecursive => {
+                write!(
+                    f,
+                    "With Mutually Recursive is not supported with the linear_chains option. \
```

Comment:
Nit: "The linear_chains option is not supported with WITH MUTUALLY RECURSIVE" seems more natural to me.